osc-server/src/main/java/org/osc/core/broker/rest/client/openstack/openstack4j/Openstack4JNeutron.java

```diff
@@ -50,7 +50,6 @@
 
     private static final String QUERY_PARAM_COMPUTE_DEVICE_OWNER = "compute:";
     private static final String QUERY_PARAM_ROUTER_DEVICE_OWNER = "network:router_interface";
-    private static final String QUERY_PARAM_TENANT_ID = "tenant_id";
     private static final String QUERY_PARAM_EXTERNAL_ROUTER = "router:external";
```

Comment:
Is router: "external"  in related to our ext-net type "external" or opnestack4j representation of ext-net?. 
because ext-net type can be default sometimes.

Reply:
its the openstack representation